src/atdf/chip.rs

```diff
@@ -65,9 +65,14 @@ pub fn parse(el: &xmltree::Element) -> crate::Result<chip::Chip> {
 
     Ok(chip::Chip {
         name: device.attr("name")?.clone(),
+        architecture: device.attr("architecture")?.clone(),
+        family: device.attr("family")?.clone(),
+        series: device.attr("series")?.clone(),
```

Comment:
Okay, found the first problem: The `series=` attribute is not always specified so we need to deal with it not being available.  You can either make it an `Option<String>` in `struct Chip` or write a default value here.  Not sure what is better.

For reference, I'm hitting these kind of errors:

```text
Error: Missing attribute "series" on
   <device architecture="AVR8" family="megaAVR" name="ATmega32U4" ...>
```

Reply:
Changed to an `Option`.